project/about.md

```diff
@@ -0,0 +1,100 @@
+sponsored by the Department of Energy ([DOE](https://www.energy.gov))
+Office of Science ([here](https://www.energy.gov/science)) Biological and
+Environmental Research ([BER](https://www.energy.gov/science/ber/biological-and-environmental-research)).
+
+Pacifica was created by EMSL and PNNL in response to 
+[DOE Data Management Policy](https://www.energy.gov/datamanagement/doe-policy-digital-research-data-management)
+to manage open research data to make it more transparent. EMSL and
+PNNL both have similar administrative problems in that a single
+project can not shoulder the burden of meeting the policy on its
+own. This brought both EMSL and PNNL to the table to collaborate on
+developing an institutional data management system for open research
+science.
+
+## The Data Management Challenge
+
+The challenge is expressed by the nexus of the operating model of a
+large institution and data policies its required to follow. 
+
+### DOE Data Management Policy
+
+The DOE Data Management policy covers a lot, but we will only highlight
+the guiding principles here.
+
+#### Effective Data Management
+
+```
+ 1. Effective data management has the potential to increase the pace of
+    scientific discovery and promote more efficient and effective use of
+    government funding and resources. Data management planning should be
+    an integral part of research planning.
+```
+
+The imparative to increase the pace of scientific discovery in 
+conjunction with being more efficient with the resources we use is
+increasingly becoming more important. Pacifica has solutions for
+these problems and would like to increase its impact in solving this
+challenge.
+
+#### Sharing Proves Integrity
+
+```
+ 2. Sharing and preserving data are central to protecting the integrity of
+    science by facilitating validation of results and to advancing science
+    by broadening the value of research data to disciplines other than the
+    originating one and to society at large. To the greatest extent, with
+    the fewest constraints possible, and consistent with the requirements
+    and other principles stated in this document, data sharing should make
+    digital research data available to and useful for the scientific
+    community, industry, and the public.
+```
+
+Sharing scientific data to the public proves the scientific work being done
+is of the highest integrity. This is another reason for making Pacifica an
+open source software project that can prove the integrity of the software
+is of the highest integrity.
+
+#### Preserve What You Can
+
+```
+ 3. Not all data need to be shared or preserved. The costs and benefits of
+    doing so should be considered in data management planning.
+```
+
+The task of sharing and preserving all data generated by open science is
+costly for small projects. The overhead of meeting the policy must then be
+addressed by the supporting institution.
+
+### The Operating Model
+
```

Comment:
Is that first statement factual across all orgs? Consider "largely project based." 

Projects have many properties, each bounded by an institution's specific needs. Properties can include: dollar amount ... etc.

Reply:
This is a good point, if you have an example institution that doesn't follow some sort of project model to manage spending of funds, I'd like to know.

Reply:
I don't have an example, but I also don't want to assume it's the same everywhere and accidentally alienate some whose workplace is somehow different.

Reply:
I added a general paragraph talking about the operating model as a function of the way research institutions spend money. The project model is a subsection of that overall section. If we find any new model that doesn't have some mapping to projects we'll find a spot for them.